index.js

```diff
@@ -18,6 +19,28 @@ const TimerMixin = require('react-timer-mixin');
 const DefaultTabBar = require('./DefaultTabBar');
 const ScrollableTabBar = require('./ScrollableTabBar');
 
+class StaticContainer extends Component{
```

Comment:
I was going to paste this as well. There is an entire subproj: https://github.com/reactjs/react-static-container. Should use that instead of you own.